client/login/magic-login/index.jsx

```diff
@@ -224,51 +264,59 @@ class MagicLogin extends Component {
 					<div className="grav-powered-magic-login__footer grav-powered-magic-login__footer--email-verification">
 						<div>{ translate( 'Are you having issues receiving it?' ) }</div>
 						<div>
-							{ translate(
-								'{{sendEmailButton}}Resend the verification email{{/sendEmailButton}} or {{showMagicLoginButton}}use a different email address{{/showMagicLoginButton}}.',
-								{
-									components: {
-										sendEmailButton: (
-											<button
-												onClick={ () =>
-													resendEmail( usernameOrEmail, {
-														redirectTo: query?.redirect_to,
-														requestLoginEmailFormFlow: true,
-														createAccount: true,
-														flow: oauth2Client.name,
-														showGlobalNotices: true,
-													} )
-												}
-												disabled={ isSendingEmail || !! resendEmailCountdown }
-											/>
-										),
-										showMagicLoginButton: (
-											<button
-												className="grav-powered-magic-login__show-magic-login"
-												onClick={ () => {
-													this.resetResendEmailCountdown();
-													showMagicLogin();
-												} }
-											/>
-										),
-									},
-								}
-							) }
+							{ resendEmailCountdown === 0
+								? translate(
+										'{{sendEmailButton}}Resend the verification email{{/sendEmailButton}} or {{showMagicLoginButton}}use a different email address{{/showMagicLoginButton}}.',
+										emailTextOptions
+								  )
+								: translate(
+										'{{showMagicLoginButton}}use a different email address{{/showMagicLoginButton}}.',
```

Comment:
Maybe make the first letter capitalised? It looks a bit strange when the "Resend" link is not there.

![image](https://github.com/Automattic/wp-calypso/assets/799065/38d55576-cd78-477b-a5ac-52bf89f282ba)

Reply:
@aaronfc Good catch, I have updated it, thanks!